wlauto/commands/create.py

```diff
@@ -392,7 +418,7 @@ def render_template(name, params):
     with open(filepath) as fh:
         text = fh.read()
         template = string.Template(text)
-        return template.substitute(params)
+        return template.safe_substitute(params)
```

Comment:
It was originally because I used parameter substitution in some of the files themselves during the build stage so when writing the template it would leave them intact, however it wouldn't be difficult to do all of the substitution here instead if that makes more sense.